frontend/src/components/Body/Tarifs/Tariffs.js

```diff
@@ -19,9 +19,17 @@ const LiComponent = styled('li')`
   padding-top: 20px;
   font-family: "Inter, sans-serif";
   font-weight: 400;
-  font-size: ${props => props.theme.breakpoints.down('sm') ? "16px" : "14px"};
+  font-size: ${props => 
+    props.theme.breakpoints.down('xs') ? "16px" :
+    props.theme.breakpoints.down('sm') ? "16px" :
+    props.theme.breakpoints.down('md') ? "18px" :
+    props.theme.breakpoints.down('lg') ? "20px" : '22px'  };
   color: #f1f1f1;
-`
+
+  @media (min-width: 1920px) { 
+    font-size: 28px; 
+  }
+`;
```

Comment:
Remove this media query. MUI have different media props: xl, lg, md, sm, xs. I think you can choose some instead it